WorkflowFunctionBundle/Document/Authorization.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace OpenOrchestra\WorkflowFunctionBundle\Document;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
+use OpenOrchestra\WorkflowFunction\Model\AuthorizationInterface;
+
+/**
+ * Description of Authorization
+ *
+ * @ODM\EmbeddedDocument
+ */
+class Authorization implements AuthorizationInterface
+{
+    /**
+     * @var string $name
+     *
+     * @ODM\Field(type="string")
+     */
+    protected $name;
+
+    /**
+     * @var ArrayCollection $workflowFunctions
+     *
+     * @ODM\ReferenceMany(targetDocument="OpenOrchestra\WorkflowFunction\Model\WorkflowFunctionInterface")
+     */
+    protected $workflowFunctions;
+
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+        $this->workflowFunctions = new ArrayCollection();
```

Comment:
Should be done in a refacto. This method is used in all entity with ArrayCollection. Create a ticket?